src/angular-experimental/journey-summary/journey-summary.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @angular-eslint/directive-selector */
+/* eslint-disable @angular-eslint/directive-selector, @angular-eslint/directive-class-suffix */
```

Comment:
I think we should configure this globally and not create exceptions for every file. @kyubisation ?

Reply:
Discussed with @DavideMininni-Fincons via Teams. We will update the eslint config.